ngraph/core/reference/include/ngraph/coordinate_index.hpp

```diff
@@ -0,0 +1,29 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+#include <cstddef>
+
+namespace ngraph // FWD
+{
+    class Coordinate;
+    class Shape;
+} // namespace ngraph
+
+namespace ngraph
```

Comment:
Why do you declare the `ngraph` namespace 2 times?

Reply:
My intention was to split stuffs which are introduce from includes/FWD. 
@ilyachur, should I join FWD with function declaration?

Reply:
Ok, I got your point.
I prefer to have one namespace but it is my subjective opinion and I am ok with current approach.

Reply:
File removed.